### PR TITLE
lxc/action: add `--all` flag

### PR DESCRIPTION
**Pull request by schu**
Allow running actions (start, stop, ...) against all containers.

Signed-off-by: Michael Schubert <michael@kinvolk.io>

### CODE REVIEW

**Comment by brauner** [comment]
Cool, thanks. I'll let @stgraber take a look at this since he was the one opening the original issue. :)

**Comment by brauner** [comment]
jenkins: test this please

**Comment by brauner** [comment]
Ah, go blowing up in Travis all over the place. Seems to be the current state of things. I think @stgraber might have already filed an issue about this against Travis.

**Comment by brauner** [comment]
@schu, the only thing that is missing is `make i18n` which updates our translation files and so on. You can just push it as an additional commit on top. I usually call it:

    lxc: make i18n

**Comment by schu** [comment]
@brauner thanks, done.

**Comment by brauner** [comment]
jenkins: test this please

**Comment by stgraber** [comment]
jenkins: test this please

**Comment by brauner** [comment]
jenkins: test this please

**Comment by brauner** [comment]
a bunch of builders went offline this time

File: lxc/action.go [hunk 1]
```diff
@@ -110,12 +112,31 @@ func (c *actionCmd) doAction(conf *config.Config, nameArg string) error {
...
+		ctslist, err := d.GetContainers()
+		if err != nil {
+			return err
+		}
+		for _, ct := range ctslist {
+			names = append(names, ct.Name)
+		}
+	} else {
+		if c.all {
+			return fmt.Errorf("Both --all and container names given")
```
Comment (stgraber) [inline lxc/action.go]: Should use i18n.G to have the string translated

Comment (schu) [reply lxc/action.go]: Fixed, thanks.

**Comment by brauner** [comment]
jenkins: test this please

**Comment by brauner** [comment]
@schu, you're missing a new `make i18n` because you've added the ` i18n.G`. Happens to me all of the time too. :)

**Comment by schu** [comment]
@brauner updated, thanks again! :)

**Comment by brauner** [comment]
jenkins: test this please

**Comment by brauner** [comment]
Thanks! :)

**Pull request merged**